presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConnectorConfig.java

```diff
@@ -0,0 +1,348 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kinesis;
+
+import io.airlift.configuration.Config;
+import io.airlift.units.Duration;
+
+import javax.validation.constraints.NotNull;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This Class handles all the configuration settings that is stored in /etc/catalog/kinesis.properties file
+ */
+public class KinesisConnectorConfig
+{
+    /**
+     * The schema name to use in the connector.
+     */
+    private String defaultSchema = "default";
+
+    /**
+     * Folder holding the JSON description files for Kinesis streams.
+     */
+    private String tableDescriptionDir = "etc/kinesis/";
+
+    /**
+     * An S3 URL with JSON description files for Kinesis streams.
+     * <p>
+     * This is empty by default and will override tableDescriptionDir when set.
+     */
+    private String tableDescriptionsS3 = "";
```

Comment:
It'd be cleaner to have a single option for this that can handle both an S3 URL and a local file. You can do that by supporting `file://` URLs and/or treat anything that doesn't start with a URL protocol segment as a local path.

Reply:
changed to this behaviour